src/js/components/mon-compte/export-credentials-component.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { InputLabel, Select, MenuItem } from '@material-ui/core';
+
+class ExportCredentialsField extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = { value: 'python' };
+		this.credentials = props.credentials;
```

Comment:
I don't think you should store the `credentials` as a class attribute.  
This component won't modify them so we can read them using `this.props.credentials`.

Reply:
Done. Change class to a function with hook (language state)